README.md

```
@@ -191,6 +238,14 @@
    - Context: Prevent CI time from creeping back up
    - Approach: GitHub Actions workflow to track CI duration trends
 
+#### Known Issues & Follow-up Items
+
+**Technical Debt (Non-User-Facing):**
+- **[Issue #118](https://github.com/TheIllusionOfLife/Eureka/issues/118)**: Coordinator evaluation parsing issues - High priority technical fix needed for coordinator command
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)